pkg/config/node.go

```diff
@@ -307,18 +309,34 @@ func getSortedBackends(kubeconfigPath string) (backends []Backend, err error) {
 	return backends, err
 }
 
-func GetLBConfig(kubeconfigPath string, apiPort, lbPort, statPort uint16, apiVip net.IP) (ApiLBConfig, error) {
+func GetLBConfig(kubeconfigPath string, apiPort, lbPort, statPort uint16, apiVip net.IP, queyrAlsoApiVipOpt ...bool) (ApiLBConfig, error) {
```

Comment:
Typo in the variable name: s/queyr/query/

Reply:
Also, can we skip the variadic workaround for Go's limitations and just always pass this? In fact, does it even need to be configurable? Are there circumstances where we need to _not_ query the api vip?